pkg/controllers/namespace/namespace_controller.go

```diff
@@ -67,6 +67,13 @@ func (r *NamespaceReconciler) createNetworkInfoCR(ctx context.Context, obj clien
 	if len(networkInfos.Items) > 0 {
 		// if there is already one networkInfo, return this networkInfo
 		log.Info("NetworkInfo already exists", "NetworkInfo", networkInfos.Items[0].Name, "Namespace", ns)
+		// Clear the error annotation on k8s ns
```

Comment:
nit: could you add some explanation here to clarify why we did this, because this scenario is quite corner, others may confuse why remove vpc error here.